libcodechecker/libhandlers/server.py

```diff
@@ -247,6 +256,30 @@ def add_arguments_to_parser(parser):
                                 help="Stops all of your running CodeChecker "
                                      "server instances.")
 
+    database_mgmnt = parser.add_argument_group(
+            "Database management arguments.",
+            """WARNING these commands needs to be called with the same
```

Comment:
The help should be better here. Do I need to first start a server, then from another shell invoke CodeChecker server --config-directory ... --postgresql ... --status?